### ISSUE TITLE
Version 0.6.2 is unpublished

### ISSUE THREAD

**Issue by mukuljainx** [dialogflow/dialogflow-fulfillment-nodejs#224]
In version 0.6.1, node version was fixed to 6 which later reverted back 0.6.2, but is not published to npm yet, hence unable to get latest version via npm.

**Comment by isikemre** [comment]
Yeah that's right. Yarn is not able to install the package because of the nodeEngine must be 6 thing, which is removed in v0.6.2. 

It would be great if you publish the 0.6.2 in npm.

**Comment by mukuljainx** [comment]
I am not authorized to publish it 😅

**Comment by isikemre** [comment]
Yeah, I know 😄
I meant the guys behind this project.

**Comment by INVITR0** [comment]
+1,000,000,000,000,000,000,000,000

**Comment by mernxl** [comment]
Please publish to enable us use the library, Yarn can't install 0.6.1 because of the node 6 issue...
